src/shared/components/Carousel/Carousel.style.ts

```diff
@@ -75,3 +49,25 @@ export const GliderContainer = styled.div<HasPadding>`
 export const Track = styled.div`
   align-items: flex-start;
 `
+
+export const Dots = styled.div`
+  margin-top: ${sizes(13)};
+
+  .glider-dot {
+    background-color: ${colors.gray[700]};
+    width: ${sizes(8)};
+    height: ${sizes(1)};
+    border-radius: 0;
+    margin: 0 ${sizes(1)};
+    transition: all ${transitions.timings.regular} ${transitions.easing};
+
+    &:hover {
```

Comment:
```suggestion
    &:hover:not(.active) {
```
This will make sure that we will not trigger the hover effect on buttons that are already active. I think it makes sense to do it that way.